app/assets/javascripts/discourse/templates/components/queued-post.hbs

```diff
@@ -20,24 +20,41 @@
     </div>
     <div class='clearfix'></div>
 
-    <span class='post-title'>
-      {{i18n "queue.topic"}}
-      {{#if post.topic}}
-        {{topic-link post.topic}}
-      {{else}}
-        {{post.post_options.title}}
-      {{/if}}
-      {{category-badge post.category}}
-    </span>
+    {{#if editTitleAndCategory}}
+      <span class="edit-title">
+        {{text-field value=buffered.title maxlength=siteSettings.max_topic_title_length}}
+      </span>
+      {{category-chooser value=buffered.category_id}}
+    {{else}}
+      <span class='post-title'>
+        {{i18n "queue.topic"}}
+        {{#if post.topic}}
+          {{topic-link post.topic}}
+        {{else}}
+          {{editables.title}}
+        {{/if}}
+        {{category-badge editables.category}}
+      </span>
+    {{/if}}
 
     <div class='body'>
       {{#if editing}}
         {{d-editor value=buffered.raw}}
       {{else}}
-        {{cook-text post.raw}}
+        {{cook-text editables.raw}}
       {{/if}}
     </div>
 
+    {{#if showTags}}
+      <div class="list-tags">
+        {{#each tags as |t|}}
+          {{discourse-tag t}}
+        {{/each}}
+      </div>
+    {{else if editTags}}
+      {{tag-chooser tags=buffered.tags categoryId=buffered.category_id width='100%'}}
```

Comment:
oh my ... lucky @jjaffeux  is working on nuking select2 completely so this kind of hackery will no longer be needed ... he will sort this out as well when he gets to replacing the tag-chooser